core/src/test/scala/unit/kafka/security/authorizer/AclAuthorizerTest.scala

```diff
@@ -82,16 +86,17 @@ class AclAuthorizerTest extends ZooKeeperTestHarness {
   override def setUp(): Unit = {
     super.setUp()
 
+    val authorizers = Seq(aclAuthorizer, aclAuthorizer2, MockAuthorizer.authorizer)
```

Comment:
The MockAuthorizer is an AclAuthorizer using the interface default to do `authorizeByResourceType`. I was trying to prevent the duplicated code and ease the test implementation. Do you think we can keep it here?